Doc/library/asyncio-eventloop.rst

```diff
@@ -188,19 +188,21 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
+   the threads in the :class:`ThreadPoolExecutor`. After calling this method,
+   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
    while using the default executor.
 
-   The *timeout* parameter specifies the amount of time the executor will
-   be given to finish joining. The default value is ``None``, which means the
-   executor will be given an unlimited amount of time.
+   The *timeout* parameter specifies the amount of time this method gives the
+   executor to finish joining. The default value is ``None``, which means the
+   method allows the executor an unlimited amount of time.
 
-   If the timeout duration is reached, a warning is emitted and executor is
-   terminated without waiting for its threads to finish joining.
+   If the timeout duration is reached, this method emits a warning and
+   terminates the default executor without waiting for its threads to finish
+   joining.
 
-   Note that there is no need to call this function when
-   :func:`asyncio.run` is used.
+   Note that there is no need to call this function when :func:`asyncio.run` is
+   used, because that high-level function handles default executor shutdown
+   automatically.
```

Comment:
```suggestion
   .. note:: There is no need to call this method when :func:`asyncio.run` is
      used, because that high-level function handles default executor shutdown
      automatically.
```

* Make this an actual note
* This is a method rather than a function, and calling it thus avoids repetition and potential confusion

Reply:
@CAM-Gerlach Is it common reST syntax to have the note text start on the same line as the `.. note::` directive? I'm pretty sure I've usually seen it always as:

```
.. note::

   There is no need ...
   ...
   ...
```

Reply:
Stylistically, I typically do

```suggestion
   .. note::
   
      There is no need to call this method when :func:`asyncio.run` is
      used, because that high-level function handles default executor shutdown
      automatically.
```

But either way produces identical rendered output, so long as the directive is configured accordingly (as the default admonitions are).